src/ast.alp

```diff
@@ -22,4 +59,32 @@ let float line val = Float {line=line, val=val}
 
 let float_val Float {line=_, val=v} = v
 
-let string line val = String {line=line, val=val}
\ No newline at end of file
+let string line val = String {line=line, val=val}
+
+let bind name line bound =
+  Binding {  line=line
+           , name=name
+           , bound=bound
+           , body=None
+           , typ=None
+          }
+
+let bind name line bound body =
+  Binding {  line=line
+           , name=name
+           , bound=bound
+           , body=Some body
+           , typ=None
+          }
+
+let bind_body Binding b_rec body = Binding {body=Some body | b_rec}
```

Comment:
This is a good point and I think there's likely value in letting the (eventual) formatter clean this sort of ambiguity up to enforce the style you're suggesting.